workflow/executor/executor.go

```diff
@@ -360,27 +362,38 @@ func (we *WorkflowExecutor) SaveParameters() error {
 	return nil
 }
 
+func (we *WorkflowExecutor) saveLogsToPath(logDir string, fileName string) (outputPath string, err error) {
```

Comment:
is this needed for grepping? 
if so, any reason why the logs couldn't be checked for matches on the fly?